core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ClusterBootstrapper.java

```diff
@@ -361,4 +373,21 @@ private void validatePollingResultForCancellation(PollingResult pollingResult, S
         }
     }
 
+    String getNodeNameForInstance(InstanceMetaData im, Stack stack,
+            Map<String, AtomicLong> hostGroupNodeIndex, Set<String> clusterNodeNames) {
+        if (isNotBlank(im.getShortHostname())) {
+            return im.getShortHostname();
+        }
```

Comment:
missing else to indicate there are multiple exit point for the method

Reply:
fixed.